.github/workflows/ci.yml

```diff
@@ -104,7 +104,9 @@ jobs:
 
     # install additional tools needed
     - name: install utils
-      run: apt-get update && apt-get install -y postgresql-client qpdf curl
+      run: apt-get update && apt-get install -y postgresql-client qpdf
```

Comment:
@infotroph did you mean to drop `curl` from the list of installs here and in the other two cases below?

Reply:
@mdietze Yes, because since #2716 curl is pre-installed on the container.

@robkooper would it make sense to add qpdf and postgresql-client to the image and then remove these apt-get calls entirely?

Reply:
only reason we have them is that the code needs them. I don't see any reason for our base image to need psql client, you can do that from R. Same for the qpdf, is just needed to make a test pass.